src/UI/Component/Tree/Node/Factory.php

```diff
@@ -41,4 +41,26 @@ interface Factory
 	 */
 	public function simple(string $label, Icon $icon=null): Simple;
 
+    /**
+     * ---
+     * description:
+     *   purpose: >
+     *     The Bylined Node is an entry containing additional information about
+     *     the node.
+     *   composition: >
+     *     It consists of a string-label, a byline an optional Icon.
+     *   effect: >
+     *     This node is a simple node with an additional string-byline.
+     * rules:
+     *   usage:
+     *      1: >
+     *        A Byline Node SHOULD when there is a need to display a byline
```

Comment:
SHOULD be used?

Reply:
That is exactly the question is it a MUST or a SHOULD do we want to enforce it?

Reply:
I did not question the SHOULD ;-). The `be used` was missing :).